test/interop/C/multilocale/SKIPIF

```diff
@@ -1,16 +1,11 @@
 #!/usr/bin/env bash
 
-# Multilocale does not work with LLVM right now.
-if [[ $CHPL_LLVM != none ]]; then
+# The dynamic tests need to run either on a Mac or with CHPL_LIB_PIC=pic
+# Otherwise they will encounter symbol relocation errors
+if [[ $CHPL_TARGET_PLATFORM == darwin || $CHPL_LIB_PIC == pic ]] ; then
+   echo False
```

Comment:
The indentation on this line is slightly different than the indentation for the else branch, I think?